src/features/simulation/components/form/util.ts

```diff
@@ -70,6 +70,12 @@ function parseMovementSimulationStrategyToRequest(
     slowDownProbability: parseInt(result.slowDownProbability) / 100,
     maxVelocity: parseInt(result.maxVelocity),
     randomProvider: 'TRUE',
+    threshold: result.type === 'BRAKE_LIGHT' ?
+      parseInt(result.threshold as string) : undefined,
+    accelerationDelayProbability: result.type === 'BRAKE_LIGHT' ?
+      parseFloat(result.accelerationDelayProbability as string) / 100 : undefined,
+    breakLightReactionProbability: result.type === 'BRAKE_LIGHT' ?
+      parseFloat(result.breakLightReactionProbability as string) / 100 : undefined,
```

Comment:
a co jak confirmniesz pustego forma, nie wydupcy sie to po stronie jsa? u mnie zrobilem funckje hasLengthAndDefined ktora sprawdzam wraz z warunkiem czy odpowiedni resukt jest zaznavczony

Reply:
nic się nie wydupca